drivers/hmis/app/graphql/types/hmis_schema/enrollment.rb

```diff
@@ -12,25 +12,30 @@ class HmisSchema::Enrollment < Types::BaseObject
     include Types::HmisSchema::HasServices
     include Types::HmisSchema::HasAssessments
 
+    def self.configuration
+      Hmis::Hud::Enrollment.hmis_configuration(version: '2022')
+    end
+
     description 'HUD Enrollment'
     field :id, ID, null: false
     field :project, Types::HmisSchema::Project, null: false
-    field :entry_date, GraphQL::Types::ISO8601Date, null: true
+    hud_field :entry_date, null: true # we are overriding HUD spec to make it nullable, why? we have it even for WIP
```

Comment:
@eanders right now we have Entry Date as nullable in the GraphQL schema. I forget if this was intentional, or if it was overlooked.. is Entry Date sometimes null in imports? For HMIS-generated data, we should always have it.

Reply:
I don't believe we'll see blank EntryDates, the importer uses it to determine if the data should be included, so an empty one would be problematic.

Reply:
Okay, I'lll go ahead and change it then.